src/components/MonthlyCalendar/readme.md

```diff
@@ -185,7 +190,358 @@ const initialState = {
+})`
+    color: ${props => props.text.main};
+    display: flex;
+    align-items: center;
+    font-size: 16px;
+    justify-content: space-between;
+`;
+
+const StyledInformationContainer = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    color: ${props => props.text.header};
+    margin-Bottom: 8px;
+    border-radius: 12px;
+    border: solid 1px #f0f0f7;
+    padding-bottom: 12px;
+`;
+
+const StyledHeaderContainer = styled.div`
+    display: flex;
+    align-items: center;
+    font-size: 16px;
+    margin: 8px 0 0 9px;
+`;
+
+const StyledHourContainer = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    color: ${props => props.text.main};
+    font-size: 14px;
+    margin-left: 11px;
+`;
+
+const StyledContentContainer = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    color: ${props => props.text.main};
+    display: flex;
+    align-items: center;
+    font-size: 14px;
+    margin-left: 62px;
+`;
+
+const StyledIconContainer = styled.div`
+    margin-right: 10px;
+`;
+
+const StyledStatusText = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    font-size: 14px;
+    color: ${props => props.text.header};
+`;
+
+function DailyTasks(props) {
+    const {
+        availableTasksCount, assignedTasksCount
+    } = props;
+
+    if (!availableTasksCount && !assignedTasksCount) return null;
+
+    return (
+        <StyledContainer>
+            {assignedTasksCount > 0 &&
+                <StyledAssignedLabel>
+                    {`${assignedTasksCount} Assigned`}
+                </StyledAssignedLabel>
+            }
+            {availableTasksCount > 0 &&
+                <StyledAvailableLabel count={availableTasksCount}>
+                    {`${availableTasksCount} Availables`}
+                </StyledAvailableLabel>
+            }
+        </StyledContainer>
+    );
+}
+
+function areDatesEqual(date1, date2) {
+    const first = new Date(date1);
+    const second = new Date(date2);
+
+    return first.getYear() === second.getYear()
+        && first.getMonth() === second.getMonth()
+        && first.getDate() === second.getDate();
+}
+
+function getAvailableTasksCountForDate(date, tasks) {
+    return tasks.filter(task => areDatesEqual(date, task.date) && !task.isAssigned).length;
+}
+
+function getAssignedTasksCountForDate(date, tasks) {
+    return tasks.filter(task => areDatesEqual(date, task.date) && task.isAssigned).length;
+}
+
+function getDrawerTitle(selectedDate) {
+    const option = { month: 'long', day: 'numeric' }
+    return selectedDate ? new Intl.DateTimeFormat( 'en-US', option).format(selectedDate) : null
+}
+
+const TasksBasicInformation = ({ count, name, title }) => count ? (
+    <StyledStatisticsContainer> 
+        {name}
+        <Badge className="rainbow-m-around_medium" label={count} title={title} />
+    </StyledStatisticsContainer> ) : null;
+
+function TaskInformation({task}) {
```

Comment:
here it could be something like that
`
const TaskInformation = ({hour, description, isConfirmed}) => (
            <StyledInformationContainer>
                    <StyledHeaderContainer>
                        <Avatar icon={<CalendarIcon />} size="medium" backgroundColor="#f4f6f9" />
                        <StyledHourContainer>{hour}</StyledHourContainer>
                    </StyledHeaderContainer>
                    <StyledContentContainer>
                        {description}
                    </StyledContentContainer>
                    {isConfirmed ?
                        <StyledContentContainer>
                            <StyledIconContainer>
                                <Avatar icon={<CheckmarkIcon />} size="x-small" backgroundColor="#1de9b6" />
                            </StyledIconContainer>
                            <StyledStatusText>Confirmed</StyledStatusText>
                        </StyledContentContainer> :
                        <StyledContentContainer>
                            <StyledStatusText> Not confirmed</StyledStatusText>
                        </StyledContentContainer>
                    }
            </StyledInformationContainer>
    )
`